src/Configuration/Utils.hs

```diff
@@ -317,18 +316,6 @@ data AppConfiguration a = AppConfiguration
     , _mainConfig ∷ !a
     }
 
--- | A flag that indicates that the application should output the effective
--- configuration and exit.
---
-printConfig ∷ Lens' (AppConfiguration a) Bool
-printConfig = lens _printConfig $ \s a → s { _printConfig = a }
-
--- | The 'ConfigFilesConfig' collects all parameters that determine how
--- configuration files are loaded and parsed.
---
-configFilesConfig ∷ Lens' (AppConfiguration a) ConfigFilesConfig
-configFilesConfig = lens _configFilesConfig $ \s a → s { _configFilesConfig = a }
```

Comment:
These weren't used anywhere, nor is their parent type `AppConfiguration` exported.